### PR TITLE
[AMS-1301] Conflict cases separated from each other by HTTP codes

### PR DESCRIPTION
**Pull request by andreychizhov**

### CODE REVIEW

**Comment by aotd1** [comment]
Нужно добавить аннотации
```
         [ProducesResponseType(423)]
```
Для корректного отображения в swagger

File: src/VStore.Host/Controllers/ObjectsController.cs [hunk 1]
```diff
@@ -250,6 +251,7 @@ public sealed class ObjectsController : VStoreController
         [ProducesResponseType(409)]
```
Comment (denisivan0v) [inline src/VStore.Host/Controllers/ObjectsController.cs]: This endpoint doesn't return 409.

File: src/VStore.Host/Controllers/TemplatesController.cs [hunk 2]
```diff
@@ -201,6 +201,7 @@ public TemplatesController(TemplatesStorageReader templatesStorageReader, Templa
         [ProducesResponseType(409)]
```
Comment (denisivan0v) [inline src/VStore.Host/Controllers/TemplatesController.cs]: This endpoint doesn't return 409. See `ModifyInternal`

File: src/VStore.Host/Controllers/TemplatesController.cs [hunk 3]
```diff
@@ -220,6 +221,7 @@ public TemplatesController(TemplatesStorageReader templatesStorageReader, Templa
...
```
Comment (denisivan0v) [inline src/VStore.Host/Controllers/TemplatesController.cs]: This endpoint doesn't return 409. See `ModifyInternal`

**Comment by andreychizhov** [comment]
Fixed swagger attributes inconsistency.

**Comment by denisivan0v** [comment]
@andreychizhov @xakep139  can we merge this PR?

**Comment by xakep139** [comment]
@denisivan0v it has to be merged after integration testing with [AMS-1300]

**Pull request merged**